### PR TITLE
Display user

### PR DESCRIPTION
**Pull request by Akxy-01**

### CODE REVIEW

**Review by bharatht96 — APPROVED** [review]
Looks good!

File: CodeBenders/Assets/Scripts/Projectile.cs [hunk 1]
```diff
@@ -31,6 +32,8 @@ public class Projectile : MonoBehaviour
     private Rigidbody2D _projectileBody;
     // the spring joint of the projectile itself
     private SpringJoint2D _projectileSpringJoint;
+    //names of players
```
Comment (late-in-autumn) [inline CodeBenders/Assets/Scripts/Projectile.cs:35]: I'd rather move this logic out to a standalone (empty) game object whose sole purpose is to update the current users (and also maybe disable firing of the other player), and only leave method calls/unity messages in this component (this comment applies to all your additions in `Projectile.cs` component).

What I can think of is that you move your player display logic out to a component that is attached to the `Display and Controls` object and make the update method public. Then in the coroutine, you can use `GameObject.SendMessage()` method to asynchronously trigger a player turn update (which may also include disable firing for the previously fired player). And each projectile can identify which player it belongs to by reading the tag that it is assosciated with (see `ProjectileGenerator.cs` for details on that).

Sorry for my software engineering hygine OCD, that's just my personal perference.

P.S. I can help refactor with this if you want, but I'm occupied with my assignment for another class now.

**Review by late-in-autumn — CHANGES_REQUESTED** [review]
See my comments for some of my thoughts.

File: CodeBenders/Assets/Scripts/Projectile.cs (hunk 1 above)
Comment (Akxy-01) [reply CodeBenders/Assets/Scripts/Projectile.cs:35]: Moving out the logic to a different component makes sense. I'll try changing it.

**Review by late-in-autumn — APPROVED** [review]
Looks cool and works for me.

**Review by bharatht96 — APPROVED** [review]

**Pull request merged**